cmake/modules/BuildArrow.cmake

```diff
@@ -102,6 +108,11 @@ function(build_arrow)
   set(arrow_BYPRODUCTS ${arrow_LIBRARY})
   list(APPEND arrow_BYPRODUCTS ${parquet_LIBRARY})
 
+  if (WITH_RADOSGW_ARROW_FLIGHT)
```

Comment:
nit, might want to drop the space after `if` and `endif` for better consistency.